systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -271,102 +273,179 @@ void testCustomAndUpdatedValues() {
         zookeeperConfig.put("tickTime", "2000");
         zookeeperConfig.put("initLimit", "5");
         zookeeperConfig.put("syncLimit", "2");
+        zookeeperConfig.put("autopurge.purgeInterval", "1");
```

Comment:
Do we really need this?

Reply:
Why not? If we can configure it and if we had checked for this parameter before, I guess we have to have it.

Reply:
Well, I do not see any harm in it. But also any particular value for this test. That is why I was wondering why was it added.